db/schema.rb

```diff
@@ -88,6 +88,19 @@
   add_index "bundle_products", ["bundle_product_id"], name: "fk_bundle_prod_prod", using: :btree
   add_index "bundle_products", ["product_id"], name: "fk_bundle_prod_bundle", using: :btree
 
+  create_table "card_readers", force: :cascade do |t|
```

Comment:
since I'm not sure yet what data is going in these, I'd like to just leave it as is until I'm sure--it should be easy enough to migrate into whatever we need at that point